README.md

```diff
@@ -146,9 +165,17 @@ You can run all docker containers and Spring applications by running the `./scri
 won't start ELK and Zipkin. If you want to use ELK and Zipkin, you can run the script as:
 `./scripts/start.sh --elk --distributed-tracing`. To stop everything you can run `./scripts/stop.sh`.
 
+Do not forget to config the hosts file:
```

Comment:
@blling Why do we have to configure the hosts file? Can we avoid this? If you don't have admin rights (like me :)) you can't actually change the hosts file.